package.json

```diff
@@ -20,19 +20,22 @@
     "snyk-protect": "snyk-protect"
   },
   "dependencies": {
+    "@chakra-ui/icons": "^1.1.1",
```

Comment:
We have `react-icons` already and another icon library seems unnecessary. Can you use icons from there?

Reply:
Sure. removed and change to use `CopyIconButton` (sorry, I didn't noticed it)